tests/unit/amo/pages/TestGuides.js

```diff
@@ -59,18 +58,43 @@ describe(__filename, () => {
   it('fetches guides addons', () => {
     const { store } = dispatchClientMetadata();
     const dispatchSpy = sinon.spy(store, 'dispatch');
-    const root = render({ dispatch: dispatchSpy, store });
+    const errorHandler = createStubErrorHandler();
+
+    render({ errorHandler, store });
 
     const content = getContent('privacy', fakeI18n());
     const guids = content.sections.map((section) => section.addonGuid);
 
+    sinon.assert.calledOnce(dispatchSpy);
+
     sinon.assert.calledWith(
```

Comment:
I believe we put the `calledOnce` after `calledWith`.

Reply:
oh okay, I thought it was the other way around but sure I'll update..